dafni/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Define the Python version neded for CTF
+FROM python:3.10-slim
+
+## Prevents Python from writing pyc files
+ENV PYTHONDONTWRITEBYTECODE=1
+#
+## Keeps Python from buffering stdout and stderr to avoid the framework
+## from crashing without emitting any logs due to buffering
+ENV PYTHONUNBUFFERED=1
+
+#Label maintainer
+LABEL maintainer="Dr. Farhad Allian - The University of Sheffield"
+
+# Copy the source code and test files from build into the container
+COPY --chown=nobody ./causal_testing /usr/src/app/
+COPY --chown=nobody ./dafni/inputs /usr/src/app/inputs/
+COPY --chown=nobody ./dafni/main_dafni.py /usr/src/app/
+
+# Change the working directory
+WORKDIR /usr/src/app/
+
+# Install core dependencies using PyPi
+RUN pip install causal-testing-framework --no-cache-dir
+
+# Use the necessaary environment variables for the script's inputs
+ENV VARIABLES=./inputs/variables.json \
```

Comment:
I understand that these are overwritten by the .env but `VARIABLES` here should be `VARIABLES_PATH`. Also, if we're assuming a .env to be supplied, are setting these within the dockerfile necessary?

Reply:
@rsomers1998 You were indeed correct! Thanks for flagging this.